tools/lsp/Cargo.toml

```diff
@@ -32,18 +32,20 @@ path = "wasm_main.rs"
 name = "slint_lsp_wasm"
 
 [features]
-backend-qt = ["slint-interpreter/backend-qt", "preview"]
+# Remove once MSRV > 1.70 (1.70 does not yet work!):
+slint = []
```

Comment:
what's that?

Because of all the rebase i couldn't find easily which PR introduced it
Why do we have a slint feautre there?